### PR TITLE
Dashboard layout

### PR DESCRIPTION
**Pull request by christiemolloy**
Adds the layout for the dashboard cards.

Mobile
Screen Shot 2020-02-20 at 10 03 29 AM

Tablet
Screen Shot 2020-02-20 at 10 03 41 AM

Desktop
Screen Shot 2020-02-20 at 10 03 53 AM

### CODE REVIEW

File: src/PresentationalComponents/Dashboard/Dashboard.js [hunk 1]
```diff
@@ -1,31 +1,110 @@
 /* eslint-disable no-console */
 import './_dashboard.scss';
-
-import { Gallery, GalleryItem } from '@patternfly/react-core/dist/js/layouts/Gallery/index';
-import { PageHeader, PageHeaderTitle } from '@red-hat-insights/insights-frontend-components/components/PageHeader';
-
+import { Card, CardHeader, CardBody, Divider, PageSection, Title } from '@patternfly/react-core';
```
Comment (AllenBW) [inline src/PresentationalComponents/Dashboard/Dashboard.js]: can yah break up these imports [kinda as seen here](https://github.com/RedHatInsights/insights-advisor-frontend/blob/d5be602bd2b7aca848507c9b0a3a735c6947e458/src/PresentationalComponents/RuleDetails/RuleDetails.js#L12) i'll make for a smaller bundle in the future

File: src/PresentationalComponents/Dashboard/Dashboard.js [hunk 2]
```diff
@@ -1,31 +1,110 @@
...
 import { Main } from '@red-hat-insights/insights-frontend-components/components/Main';
 import React from 'react';
-import asyncComponent from '../../Utilities/skeletonAsyncCard';
-
-const ComplianceCard = asyncComponent(() => import('../../SmartComponents/Compliance/ComplianceCard'));
-const VulnerabilityCard = asyncComponent(() => import('../../SmartComponents/Vulnerability/VulnerabilityCard'));
+// import asyncComponent from '../../Utilities/skeletonAsyncCard';
```
Comment (AllenBW) [inline src/PresentationalComponents/Dashboard/Dashboard.js]: Rather than commenting this out, why not just plop em in where they'll be going eventually?

**Comment by kybaker** [comment]
@christiemolloy This looks great thank you.

**Comment by AllenBW** [comment]
as promised, DA CONFLICTS 🙃 🤗 resolve em n lets get this sucker merged!

File: src/PresentationalComponents/Dashboard/Dashboard.js (hunk 2 above)
Comment (christiemolloy) [reply src/PresentationalComponents/Dashboard/Dashboard.js]: Yep great idea!

**Comment by christiemolloy** [comment]
Thanks for the review @AllenBW updated!

File: src/PresentationalComponents/Dashboard/Dashboard.js (hunk 1 above)
Comment (AllenBW) [reply src/PresentationalComponents/Dashboard/Dashboard.js]: still this one @christiemolloy  🍍

Comment (christiemolloy) [reply src/PresentationalComponents/Dashboard/Dashboard.js]: Woops! Just updated

File: src/PresentationalComponents/Dashboard/Dashboard.js [hunk 3]
```diff
@@ -1,9 +1,11 @@
 /* eslint-disable no-console */
 import './_dashboard.scss';
-
-import { Gallery, GalleryItem } from '@patternfly/react-core/dist/js/layouts/Gallery/index';
-import { PageHeader, PageHeaderTitle } from '@red-hat-insights/insights-frontend-components/components/PageHeader';
-
+import { Card } from '@patternfly/react-core';
```
Comment (AllenBW) [inline src/PresentationalComponents/Dashboard/Dashboard.js]: almost kinda, we gotta pull each one from the dist... so card would be `import { Card } from '@patternfly/react-core/dist/js/components/Card/Card';`

Comment (christiemolloy) [reply src/PresentationalComponents/Dashboard/Dashboard.js]: ah! ok tried again

**Pull request merged**